Data-Structures/src/test/java/graph/GraphTest.java

```diff
@@ -182,4 +182,93 @@ public void testAddEdgeBetweenNodesThatHaveNotAddedToGraph() {
         graph.addEdge(node3, node1);
     }
 
+    ///////////////////////////////////// Breadth First Traversal Test //////////////////////////////////////////////////
+
+    // Test for normal case like the example in the challenge description
+    @Test
+    public void testBreadthFirstNormalCaseWhereMostNodesAreConnectedBiDirectionalToEachOther() {
+        Node pandora = graph.addNode("Pandora");
+        Node arendelle = graph.addNode("Arendelle");
+        Node metroville = graph.addNode("Metroville");
+        Node monstropolis = graph.addNode("Monstropolis");
+        Node narnia = graph.addNode("Narnia");
+        Node naboo = graph.addNode("Naboo");

```

Comment:
When you make these nodes, don't forget to declare their type using the diamond operator, like `Node<String> pandora`